<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>700</width>
      <height>630</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>-1</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>请输入文法或源程序：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_help">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>271</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_state">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>590</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>550</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>运行状态：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParse">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Parse</string>
    </property>
    <addaction name="action_parse"/>
   </widget>
   <widget class="QMenu" name="menuDFA">
    <property name="title">
     <string>DFA</string>
    </property>
    <addaction name="action_DFA"/>
   </widget>
   <addaction name="menuParse"/>
   <addaction name="menuDFA"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_parse">
   <property name="text">
    <string>语法分析</string>
   </property>
  </action>
  <action name="action_DFA">
   <property name="text">
    <string>文法转DFA</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
